ui/components/component-library/avatar-favicon/avatar-favicon.types.ts

```diff
@@ -12,7 +12,7 @@ export enum AvatarFaviconSize {
 }
 
 export interface AvatarFaviconStyleUtilityProps
-  extends Omit<AvatarBaseStyleUtilityProps, 'size'> {
+  extends Omit<AvatarBaseStyleUtilityProps, 'size' | 'children'> {
```

Comment:
Omitting children as we don't use allow `children` prop to be used in `AvatarFavicon`